website/package.json

```diff
@@ -25,7 +25,8 @@
     "@babel/plugin-syntax-import-attributes": "^7.23.3",
     "@testing-library/react": "^14.1.2",
     "@types/chai": "^4.3.11",
-    "@types/mocha": "^10.0.5",
+    "@types/jest": "^29.5.11",
```

Comment:
why is jest here? We are running tests with mocha and chai

Reply:
I am not sure how it got there and don't know what it does.